pubspec.lock

```diff
@@ -7,49 +7,42 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.13"
+    version: "2.0.11"
```

Comment:
Upgrading or downgrading pub is unrelated to your PR.
You should provide description for these changes in your PR.

Besides, use flutter channel master for running this application.